src/test/java/com/endava/internship/service/UserServiceImplTest.java

```diff
@@ -98,23 +98,22 @@ void shouldReturnGroupedMapByNumberOfPrivileges() {
 
     @Test
     void shouldReturnAverageAgeForUsers() {
-        final int expectedAverage = 23;
+        Double value = Double.valueOf(23);
+        final Optional<Double> expectedAverage = Optional.of(value);
```

Comment:
`final Optional<Double> expectedAverage = Optional.of(23.0);`

Reply:
done